### ISSUE TITLE
Accordion Chrome-specific Browser Bug

### ISSUE THREAD

**Issue by SMaguina** [twbs/bootstrap#17960]
There is a browser bug that I found occurring within Chrome and have had since March of this year (2015). I reported it to Google Chrome at that time, but to no avail with a fix. I have ran HTML Validation and Lint tests on my [one-page website](http://task-ly.netlify.com) successfully.. As you can see at the bottom page, the accordion produces multiple lines of text overlay inside the panel that is open. It may take a few panel clicks before the bug appears. 

![Panel Bug]
(http://i1146.photobucket.com/albums/o531/smag23/panel_zpsyyueochc.png)

Below are the specs on the browser/OS environment this bug occurs in:
<b>
Chrome Version 46.0.2490.71 (64-bit)
OSX Yosemite Version 10.10.5
</b>

The accordion loads fine on Firefox and Safari with the OSX specs above. In an effort to find a potential solution, I've tested it on other devices as well, such as Internet Explorer and mobile devices. The accordion works fine on Internet Explorer 11 on Windows 7, Safari browser iOS 9.0.2 on iPhone 6, and Chrome browser Android 5.0.1 on LG G3.

I've also tried isolating the problem on Codepen, however, the accordion does not dynamically operate (panels remain static) upon importing my original code and the Bootstrap accordion documentation code. I have a repo of the source code [here](https://github.com/SMaguina/Task.ly/blob/master/index.html) for this particular site in case that helps with seeing the bigger picture. 

Again, I have tested this on several browsers and devices with the accordion displaying successfully without the multiple text overlay. It is specifically with Google Chrome with the specs bolded above that this bug appears in. My best guess is it may be caused due to having several multiple accordion panels stacked (10 to be exact) and the Chrome browser having trouble rendering the text for each one all at once upon executing the accordion collapse.

**Comment by cvrebert** [comment]
> I reported it to Google Chrome at that time, but to no avail with a fix.

Link to the Chromium issue please?

**Comment by patrickhlauke** [comment]
i may be wrong. but i suspect - from enabling/disabling some CSS - that
```
.panel-body:after, .panel-body:before, {
  ...
  display: table;
  ...
}
```
has something to do with it...

**Comment by twbs-closer** [comment]
Hey there!

We're automatically closing this issue since the original poster (or another commenter) hasn't yet responded to the question or request made to them 14 days ago. We therefore assume that the user has lost interest or resolved the problem on their own. Closed issues that remain inactive for a long period may get automatically locked.

Don't worry though; if this is in error, let us know with a comment and we'll be happy to reopen the issue.

Thanks!

(*Please note that this is an [automated](https://github.com/twbs/no-carrier) comment.*)

**Comment by SMaguina** [comment]
Sorry for the late reply here.

The issue was reported using Chrome under Settings > About > Report an issue. There was no email record received or sent to me after reporting it though. I'll keep Chromium in mind next time for any Chrome Browser bug specific issues, and I will go ahead and post there. 

Just clarifying, the 'bootstrap.css' file (using latest version v3.3.5) was not modified by me in any way. As patrick suggested above, I have went into the file to enable/disable the CSS in those lines. It seems to work when I disable/comment out:  `display: table;`

However, it does mess with the overall format of the homepage and double bolds the text (without the multiple lines of text overlay). So it is a partial fix and not perfect, but it did effect how the bug is displaying which is somewhat better than before.
<b>
 ![Panel Bug]
(http://i1146.photobucket.com/albums/o531/smag23/displaytableremoved_zps3uwmyhkx.png)

I have refactored my styles.css file and removed any unused selectors and properties. To isolate the problem more, using the <a href="http://jigsaw.w3.org/css-validator/#validate_by_upload">W3C CSS Validation site</a>, I 'Validated by file upload' the styles.css and the only errors showing were font related parse errors that I cleared up. So there are no errors in both the HTML and CSS validation tests, and the bug is still appearing. Next step is posting in Chromium. Thanks.
